doc/kafka-playground/bin/produce-event

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+require_relative '../config/environment'
+
+class ProduceTopic < Thor
+  default_command :produce
+
+  desc 'TOPIC FILE...', 'produce a new event at a given Apache Kafka topic'
+  option :partition, aliases: '-p', type: :numeric,
+         desc: 'The topic partitions to write to'
+  option :partition_key, aliases: '-k', type: :string,
+         desc: 'The partition key to use to select the partition'
+  option :verbose, aliases: '-v', type: :boolean,
+         desc: 'Enable verbose outputs'
+  def produce(topic, *files)
+    debug! options
+
+    opts = {
+      topic: topic,
+      partition: options[:partition]&.to_i,
+      partition_key: options[:partition_key]
+    }.compact
+
+    if options.key?(:partition) && options.key?(:partition_key)
+      STDERR.puts 'Either use the fixed partition or a partition key.'
+      STDERR.puts 'But not both together.'
+      exit 1
+    end
+
+    files = files.map do |file|
+      next '/dev/stdin' if file == '-'
+
+      unless File.file? file
+        STDERR.puts "File '#{file}' does not exist."
+        next
+      end
+
+      file
+    end.compact.uniq
+
+    if files.empty?
+      STDERR.puts 'No files given or exists.'
+      STDERR.puts 'You have to specify file(s) or use `-\' for stdin.'
```

Comment:
It's not unusual to use this format on messages. Eg.:

```
gcc: consider using `-pg' instead of `-p' with gprof (1)
gnupg2: `%s' ist keine JPEG-Datei
grub: the size of `%s' is too large
gawk: `--posix' overrides `--characters-as-bytes'
libc: failed to add file watch `%s`: %s     
```

(`grep -H -aoP "[\`']([^\`'])+." -R /usr/share/locale/de | grep -a '`' | grep -a "'"`)